### PR TITLE
Adjust tests of triangular solve rules to new EnzymeTestUtils capabilities

### PR DESCRIPTION
**Pull request by simsurace**

### CODE REVIEW

**Review by wsmoses — APPROVED** [review]

**Comment by wsmoses** [comment]
@simsurace is there anything here you're waiting on before merging?

**Comment by simsurace** [comment]
I don't have merge rights! :)

**Pull request merged**
